build.fsx

```diff
@@ -3,13 +3,14 @@
 // And to release do: dotnet fake run build.fsx -e configuration=Release -t Release
 
 #r "paket:
+source release/temp-recursion-break
 source release/dotnetcore
 source https://api.nuget.org/v3/index.json
 nuget FSharp.Core
-nuget Microsoft.Build 17.3.2
-nuget Microsoft.Build.Framework 17.3.2
-nuget Microsoft.Build.Tasks.Core 17.3.2
-nuget Microsoft.Build.Utilities.Core 17.3.2
+nuget Microsoft.Build 17.11.4
```

Comment:
Question: Does it still need to reference all 4 of these Microsoft.Build packages here?

Older versions of MSBuild.StructuredLogger used to depend on all of these, but newer versions (including the 2.1.815 build currently used by Fake) only depend on ```Framework``` and ```Utilities.Core``` so I wonder if the base one and ```Tasks.Core``` can be removed now?